101-aci-dynamicsnav/azuredeploy.json

```diff
@@ -0,0 +1,150 @@
+{
+    "$schema": "https://schema.management.azure.com/schemas/2015-01-01/deploymentTemplate.json#",
+    "contentVersion": "1.0.0.0",
+    "parameters": {    
+      "name": {
+          "type": "string",
+          "metadata": {
+              "description": "Name for the container group"
+          },
+          "defaultValue": "acinavcontainergroup"
+      },    
+      "navRelease": {
+          "type": "string",
+          "metadata": {
+              "description": "After the colon: Identifier for the NAV release like 2017-cu11-de or devpreview-finus (always keep 'microsoft/dynamics-nav:')"
+          },
+          "defaultValue": "microsoft/dynamics-nav:2017"
+      },    
+      "username": {        
+          "type": "string",
+          "metadata": {
+              "description": "Username for your NAV super user"
+          },
+          "defaultValue": "admin"
```

Comment:
remove defaultValue, it increases surface area

Reply:
Wouldn't it make sense to have a meaningful default, so users don't have to enter something if the default works for them?

Reply:
not for credentials, they should supply their own...

Reply:
done